### DISCUSSION TITLE
Is there an option for multiple profile logins (Account switching) on web?

### DISCUSSION THREAD

**Discussion by Andre601** [mastodon/mastodon#20251] [Q&A]
Right now I have two accounts made on the same instance and I would like to be able to switch between them whenever possible (Similar to how you can do that on twitter)

Is Mastodon offering such a feature? The instance is currently using `v4.0.0rc2+glitch` of Mastodon and I'm currently using the Advanced mode.

If not, I'll consider opening an issue (If not already present) to suggest such an addition. Perhaps the advanced mode, which looks similar to Twitter's Tweetdeck, could then have an extra feature to allow defining columns of different accounts?

I feel like this is a must have as people may have multiple accounts (i.e. a personal one and one for a group, company, etc.) and switching between them should be an easy and accessible experience (Also, mobile has this feature too, so why not web/desktop?)

**Comment by asopao** [comment]
I have the same problem. I do all my social media work (professional) on the web on a laptop. How do we encourage Mastodon developers to add the toggle that is apparently include on the phone apps?

**Comment by trwnh** [comment]
- #380
